### PR TITLE
update: added confirmation modal to items in album

### PR DESCRIPTION
**Pull request by brian-david**
There is a lot of css customisation on the other components, so I needed to use !important in a lot of places to get around the other styling, especially for buttons

### CODE REVIEW

File: src/components/Album/Item.tsx [hunk 1]
```diff
@@ -14,19 +16,28 @@ interface ItemProps {
 const Item: React.FC<ItemProps> = props => {
   const item = props.imageRecord
   const { dynamoDbItems, setDynamoDBItems } = props
-
   const { filepath } = item
-
+  const [confirmModalOpen, setConfirmModalOpen] = useState(false)
   const deleteItem = () => {
+    setConfirmModalOpen(false)
```
Comment (ksamson2) [inline src/components/Album/Item.tsx]: If it's set to false by default, why do we need this line?

File: src/components/ConfirmModal/confirmModal.scss [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+button.is-normal {
+  background-color: white !important;
+  color: black !important;
+  :hover {
+    color: black !important;
+  }
+}
+
+button.is-normal :hover {
```
Comment (ksamson2) [inline src/components/ConfirmModal/confirmModal.scss]: On my end, this style doesn't take effect so the text is white and not visible

**Review by ksamson2 — CHANGES_REQUESTED** [review]
I like the overall look, I think the buttons are a bit too big and the modal is a bit squashed. Either later tonight or early tomorrow, Nicole and I should merge in new updates to the album page. This moves the delete button to the edit page. Sorry for conflict, but could you update your PR after that is merged in so that this happens on the edit page then and not on the album screen?

**Comment by ksamson2** [comment]
I think sync with Christoff because I know he fixed getting around using the !important recently. Could we not maybe attach a class to the button so that we aren't relying on our or even bulma's default styles for the button?

**Comment by brian-david** [comment]
> I like the overall look, I think the buttons are a bit too big and the modal is a bit squashed. Either later tonight or early tomorrow, Nicole and I should merge in new updates to the album page. This moves the delete button to the edit page. Sorry for conflict, but could you update your PR after that is merged in so that this happens on the edit page then and not on the album screen?

**Comment by brian-david** [comment]
> I like the overall look, I think the buttons are a bit too big and the modal is a bit squashed. Either later tonight or early tomorrow, Nicole and I should merge in new updates to the album page. This moves the delete button to the edit page. Sorry for conflict, but could you update your PR after that is merged in so that this happens on the edit page then and not on the album screen?

yip, I'll check in tonight and update

**Comment by brian-david** [comment]
> > I like the overall look, I think the buttons are a bit too big and the modal is a bit squashed. Either later tonight or early tomorrow, Nicole and I should merge in new updates to the album page. This moves the delete button to the edit page. Sorry for conflict, but could you update your PR after that is merged in so that this happens on the edit page then and not on the album screen?

I expanded the modal a bit. I thought it made sense to have big button as we're going for old blind(ish) people. But I can make them smaller

**Comment by brian-david** [comment]
Oh and there shouldn't (touch wood) be any merge conflicts now if you pull into dev

**Review by christoff-linde — APPROVED** [review]

**Pull request merged**